lib/custom-logger.js

```diff
@@ -26,15 +30,82 @@ function defaultRedactionRules() {
   }
 }
 
+function timestampFunction() {
+  return `,"time":${Date.now() / 1000.0}`
```

Comment:
Please copy from here commenting:

https://github.com/fastify/fastify/blob/8b8485ac5ff56d1ead387f6dd76384f996a2cf8e/lib/logger.js#L71

Reply:
Sorry I don't get what you mean, instead of using `Date.now() / 1000.0` use the `now` function you linked?